io/io.go

```diff
@@ -1,248 +1,279 @@
-// Licensed to the Apache Software Foundation (ASF) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The ASF licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
-// with the License.  You may obtain a copy of the License at
```

Comment:
this is replacing the entire file so it's hard to see the diffs, can you make sure it didn't change the line endings or something?